### DISCUSSION TITLE
What is the source of the error when using the `npm install` command?

### DISCUSSION THREAD

**Discussion by subnwa** [npm/feedback#631] [CLI] [answered]
While trying to load the modules, a problem arises as follows. All I do is use the `npm install` command.

I have the same problem when using the `npm login` command. I want to solve the source of this problem

![resim](https://user-images.githubusercontent.com/68991507/150686257-54cf21a1-14fc-4452-bf59-e64eef56e378.png)

**Comment by ljharb** [comment]


**Reply by subnwa** [reply]
```
{
  npm: '8.1.4',
}
```

I am using this version.

**Reply by subnwa** [reply]
`node: '17.2.0',`

**Reply by subnwa** [reply]
`openssl: '3.0.0+quic'`

also my version of openssl.

**Reply by subnwa** [reply]
no i installed normally

**Comment by MylesBorins** [comment, marked as answer]
This repo is intended for feedback + feature requests for npm, not for general technical support. You can try opening a thread over at https://github.community or ask over at stack overflow

**Reply by subnwa** [reply]
thanks
